### ISSUE TITLE
[FI] Double vowels in certain words are heard as single.

### ISSUE THREAD

**Issue by salleq** [home-assistant/intents#2480]
Saying:
"Millainen sää on"

Response:
"Pahoittelut, en löydä laitetta tai kohdetta nimellä sä on"

It drops the second 'ä' even though i extend the 'äääää' however comically long.

**Comment by v1k70rk4** [comment]
Hi, unfortunately, this happens in Hungarian as well. Microsoft STT works quite well for our language, but the others tend to guess when it comes to double letters. For example, in Hungarian, it only affects two words, but I marked one of the letters as optional, like sä[ä].  

Our language already has comically many variations due to the conjugation rules :) This fits right in.

**Comment by sappafrancesco** [comment]
Agreed with your own read on this: hassil only ever sees whatever text the speech-to-text engine hands it, there's no vowel-dropping happening in the sentence matching or the intents data. If "sää" is consistently coming through as "sä", that's the STT transcription step, not something fixable in this repository. Worth reporting against whichever STT engine is in use (Whisper, faster-whisper, etc.) with a debug trace showing the raw transcription.
